QEMU build issue - INT64_C() not declared

When building QEMU on Mac OS 10.4.11, I saw this error message: 
/private/tmp/qemu20170423-13118-3m9tys/qemu-2.4.0.1/disas/libvixl/utils.h:47:28: error: 'INT64_C' was not declared in this scope
   int64_t limit = INT64_C(1) << (n - 1);

I have gcc 5.2 installed on my system.

--- a/Library/Formula/qemu.rb
+++ b/Library/Formula/qemu.rb
@@ -52,10 +52,16 @@ class Qemu < Formula
       --prefix=#{prefix}
       --cc=#{ENV.cc}
       --host-cc=#{ENV.cc}
-      --enable-cocoa
       --disable-bsd-user
       --disable-guest-agent
     ]
+
+    # Cocoa UI uses features that require 10.5 or newer
+    if MacOS.version > :tiger
+      args << "--enable-cocoa"
+    else
+      args << "--disable-cocoa"
+    end
 
     # qemu will try to build 64-bit on 64-bit hardware, but we might not want that
     args << "--cpu=#{Hardware::CPU.arch_32_bit}" unless MacOS.prefer_64_bit?

--- a/Library/Formula/qemu.rb
+++ b/Library/Formula/qemu.rb
@@ -45,8 +45,13 @@ class Qemu < Formula
   def install
     ENV["LIBTOOL"] = "glibtool"
 
-    # Needed for certain stdint macros on 10.4
-    ENV.append_to_cflags "-D__STDC_CONSTANT_MACROS" if MacOS.version < :leopard
+    if MacOS.version < :leopard
+      # Needed for certain stdint macros on 10.4
+      ENV.append_to_cflags "-D__STDC_CONSTANT_MACROS"
+
+      # Rez is provided in a normal path in later Xcodes
+      ENV.prepend_path "PATH", "/Developer/Tools"
+    end
 
     args = %W[
       --prefix=#{prefix}

--- a/Library/Formula/qemu.rb
+++ b/Library/Formula/qemu.rb
@@ -26,6 +26,7 @@ class Qemu < Formula
   depends_on "jpeg"
   depends_on "gnutls"
   depends_on "glib"
+  depends_on "libutil" if MacOS.version < :leopard
   depends_on "pixman"
   depends_on "vde" => :optional
   depends_on "sdl" => :optional

--- a/Library/Formula/qemu.rb
+++ b/Library/Formula/qemu.rb
@@ -45,6 +45,9 @@ class Qemu < Formula
   def install
     ENV["LIBTOOL"] = "glibtool"
 
+    # Needed for certain stdint macros on 10.4
+    ENV.append_to_cflags "-D__STDC_CONSTANT_MACROS" if MacOS.version < :leopard
+
     args = %W[
       --prefix=#{prefix}
       --cc=#{ENV.cc}

--- a/Library/Formula/qemu.rb
+++ b/Library/Formula/qemu.rb
@@ -21,6 +21,7 @@ class Qemu < Formula
     sha256 "043f1c5b577fdfbaac516bc1ca909dee089cd591fd018b68c1298ab859170ef9" => :mavericks
   end
 
+  depends_on "make" => :build if MacOS.version < :leopard
   depends_on "pkg-config" => :build
   depends_on "libtool" => :build
   depends_on "jpeg"
@@ -51,6 +52,10 @@ class Qemu < Formula
 
       # Rez is provided in a normal path in later Xcodes
       ENV.prepend_path "PATH", "/Developer/Tools"
+
+      # Make 3.80 does not support the `or` operator and has trouble evaluating `unnest-vars`
+      # See https://github.com/mistydemeo/tigerbrew/pull/496
+      ENV["MAKE"] = make_path
     end
 
     args = %W[
@@ -76,7 +81,7 @@ class Qemu < Formula
     args << (build.with?("libssh2") ? "--enable-libssh2" : "--disable-libssh2")
 
     system "./configure", *args
-    system "make", "V=1", "install"
+    make "V=1", "install"
   end
 
   test do

